pkg/daemons/agent/agent.go

```diff
@@ -170,20 +165,7 @@ func startKubelet(cfg *config.Agent) error {
 	}
 
 	if cfg.Rootless {
-		// "/sys/fs/cgroup" is namespaced
-		cgroupfsWritable := unix.Access("/sys/fs/cgroup", unix.W_OK) == nil
-		if hasCFS && hasPIDs && cgroupfsWritable {
-			logrus.Info("cgroup v2 controllers are delegated for rootless.")
-			// cgroupfs v2, delegated for rootless by systemd
-			argsMap["cgroup-driver"] = "cgroupfs"
-		} else {
-			logrus.Warn("cgroup v2 controllers are not delegated for rootless. Setting cgroup driver to \"none\".")
-			// flags are from https://github.com/rootless-containers/usernetes/blob/v20190826.0/boot/kubelet.sh
-			argsMap["cgroup-driver"] = "none"
-			argsMap["feature-gates=SupportNoneCgroupDriver"] = "true"
-			argsMap["cgroups-per-qos"] = "false"
-			argsMap["enforce-node-allocatable"] = ""
-		}
+		createRootlessConfig(argsMap, hasCFS, hasCFS)
```

Comment:
Perhaps, we can do:
```go
if cfg.Rootless && `runtime.GOOS != "windows" {
    createRootlessConfig(argsMap, hasCFS, hasCFS)
}
```
This will allow us to remove the blank function below.

Reply:
See my response above about convention.

Reply:
Resolving based on other conversation